index.html

```diff
@@ -2409,6 +2409,45 @@ <h4>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Client Hints
+      </h2>
+      <p>
+        Developers often need to know details about how their web app is being
+        rendered in order to make content-negotiation decisions. Though this
+        information is accessible via JavaScript, when sent as part of a
+        {{Request}}, servers are empowered to do the content-negotiation
+        earlier in the process of setting up the web app, before JavaScript
+        execution takes place. To convey this information, this document
+        provides for the following <a data-cite=
+        "client-hints-infrastructure#client-hint-token">client hint token</a>:
+      </p>
+      <ul>
+        <li>`Sec-CH-Display-Mode` whose value is a suitable [=display modes
+        value=]
+        </li>
+      </ul>
+      <section>
+        <h3>
+          The 'Sec-CH-Display-Mode' Header Field
```

Comment:
Why "Sec-"? I presume "Sec-" stands for "Security", but I'm not sure how display mode relates to security? Maybe it should just be `CH-Display-Mode` or is that not how client hints are specified?

Reply:
`Sec-CH-*` is the common prefix for such hints sent by the user agent over secure connections (display mode being `standalone` depends on the PWA to be installed). Also see [User Preference Media Features Client Hints Headers](https://tomayac.github.io/user-preference-media-features-headers/) that we're specifying at the moment.